Lib/uuid.py

```diff
@@ -601,6 +632,22 @@ def _generate_time_safe():
                     _uuid_generate_time(_buffer)
                     return bytes(_buffer.raw), None
                 break
+        # if _uuid_generate_time has not been found (Linux) then we try libc
+        # as libc is already dynamically linked (or found above) verify a valid
+        # lib value, then look for uuid_generate()
+        if not lib:
+            try:
+                lib = ctypes.CDLL(None)
+            except:
+                lib = None
+        if lib:
+            if hasattr(lib, 'uuid_create'):    # pragma: nocover
+                _uuid_generate_time = lib.uuid_create
+                def _generate_time_safe():
+                    _buffer = ctypes.create_string_buffer(16)# uuid
+                    _status = ctypes.create_string_buffer(2) # c_ushort
+                    _uuid_generate_time(_buffer, _status)
+                    return bytes(_buffer.raw), bytes(_status.raw)
```

Comment:
This only searches the last found lib, whatever it happens to be.

Better search `uuid_create` with other exports and save in a temporary var if found. After the loop, execute this code if `uuid_create` is found and others aren't.